src/transformers/modeling_longformer.py

```diff
@@ -440,13 +431,6 @@ def forward(
             +ve: global attention
 
         """
-        attention_mask = attention_mask.squeeze(dim=2).squeeze(dim=1)
```

Comment:
This actually does not have to be re-computed for every layer, so I moved it to the `LongformerEncoder`. It could give a (very) small speed-up